src/test/java/net/sf/jabref/exporter/layout/format/DefaultTest.java

```diff
@@ -0,0 +1,37 @@
+package net.sf.jabref.exporter.layout.format;
+
+import net.sf.jabref.exporter.layout.ParamLayoutFormatter;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DefaultTest {
+
+    @Test
+    public void testSimpleText() {
+        ParamLayoutFormatter a = new Default();
```

Comment:
I would use here Default as the type instead of ParamLayoutFormatter. This makes is very immediate which class is tested...
Applies also to other tests.